### PR TITLE
Added a docker file for presentation viewer

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
- Added a docker file to build image for presentation viewer
- Added server configuration

Merge this PR after https://github.com/JankariTech/web-app-presentation-viewer/pull/9

Related issue: 
- https://github.com/JankariTech/web-app-presentation-viewer/issues/12

### CODE REVIEW

**Comment by koebel** [comment]
Usually the `web.config.json` is not part of the web extension. If this file only serves as an example to be added to whereever you are running ocis web, it's better to call it something like `web.config.json.sample`, similar to how this is done in ocis web, see https://github.com/owncloud/web/blob/master/config/config.json.sample-ocis

**Comment by individual-it** [comment]
fixed the config file issue in #9

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM owncloudops/nginx
+
+RUN rm -f /var/lib/nginx/html/*
+
+ADD docker/vhost.conf /etc/nginx/vhost.conf
+ADD dist /var/lib/nginx/html
```
Comment (saw-jan) [inline Dockerfile:6]: should dockerfile be able to build the extension with `docker build` command? That should prevent the extra build extension command before running `docker build`

**Review by koebel — APPROVED** [review]
LGTM, however please get this also reviewed from someone in the team who has more experience with docker :)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+FROM owncloudops/nginx
+
+WORKDIR /var/lib/nginx/html
+
+RUN rm -f /var/lib/nginx/html/*
+
+ADD docker/vhost.conf /etc/nginx/vhost.conf
+COPY --from=BUILD_STAGE /extension/dist /var/lib/nginx/html
+
+EXPOSE 8082
```
Comment (saw-jan) [inline Dockerfile:20]: ngix is configure to run on `8080`
```suggestion
EXPOSE 8080
```

**Review by saw-jan — APPROVED** [review]
LGTM

**Pull request merged**